packages/components/molecules/f-skeleton-loader/src/components/skeletons/RestaurantCardHeader.vue

```diff
@@ -1,9 +1,9 @@
 <template>
-    <div :class="$style['c-carouselCardRestaurantHeader']">
-        <image-block :class="$style['c-carouselCardRestaurantHeader-logo']" />
-        <heading :class="$style['c-carouselCardRestaurantHeader-heading']" />
-        <text-block :class="[$style['c-text']]" />
-        <text-block :class="[$style['c-text']]" />
+    <div :class="$style['c-restaurantCardHeader']">
+        <image-block :class="$style['c-skeletonLoader-logo']" />
+        <heading :class="$style['c-skeletonLoader-heading']" />
+        <text-block :class="[$style['c-skeletonLoader-text']]" />
+        <text-block :class="[$style['c-skeletonLoader-text']]" />
```

Comment:
I'd only prefix classnames with `c-skeletonLoader` if the component can be used anywhere (so the `c-sentence` one for instance).

As this component is hard linked to the `restaurantCard`, makes sense for these to be `c-restaurantCardHeader`

```suggestion
    <div :class="$style['c-restaurantCardHeader']">
        <image-block :class="$style['c-restaurantCardHeader-logo']" />
        <heading :class="$style['c-restaurantCardHeader-heading']" />
        <text-block :class="[$style['c-restaurantCardHeader-text']]" />
        <text-block :class="[$style['c-restaurantCardHeader-text']]" />
```